src/groupmems.c

```diff
@@ -127,7 +127,7 @@ static void add_user (const char *user,
 			static struct sgrp sgrent;
 			sgrent.sg_name = xstrdup (newgrp->gr_name);
 			sgrent.sg_mem = dup_list (newgrp->gr_mem);
-			sgrent.sg_adm = XMALLOC(1, char *);
+			sgrent.sg_adm = XMALLOC(2, char *);
```

Comment:
Hm, that's only the case for FIRST_MEMBER_IS_ADMIN.  I don't see a way to set that through configure, anyway.  I wonder if that's obsolete.

Reply:
Hmmm, good catch.

```c
$ grep -rn FIRST_MEMBER_IS_ADMIN
ChangeLog:7456:	* src/gpasswd.c: Move comment regarding FIRST_MEMBER_IS_ADMIN to
ChangeLog:7458:	* src/gpasswd.c: Indicate the end of the #ifdef FIRST_MEMBER_IS_ADMIN
src/gpasswd.c:700:#if !defined(FIRST_MEMBER_IS_ADMIN)
src/gpasswd.c:809:#ifdef FIRST_MEMBER_IS_ADMIN
src/groupmems.c:133:#ifdef FIRST_MEMBER_IS_ADMIN
src/groupmems.c:216:#ifdef FIRST_MEMBER_IS_ADMIN
NEWS:1969:  (define FIRST_MEMBER_IS_ADMIN to get the old gpasswd behaviour)
```

I'll get rid of that dead code.